language/tools/move-cli/src/utils/movey_credential.rs

```diff
@@ -0,0 +1,184 @@
+// Copyright (c) The Diem Core Contributors
+// Copyright (c) The Move Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::{bail, Context, Result};
+use move_command_line_common::movey_constants::MOVEY_URL;
+use std::fs;
+use toml_edit::easy::Value;
+
+pub const MOVEY_CREDENTIAL_PATH: &str = "/movey_credential.toml";
+
+pub fn get_registry_api_token(move_home: &str) -> Result<String> {
+    if let Ok(content) = get_api_token(&move_home) {
+        Ok(content)
+    } else {
+        bail!(
+            "There seems to be an error with your Movey API token. \
+            Please run `move movey-login` and follow the instructions."
+        )
+    }
+}
+
+pub fn get_api_token(move_home: &str) -> Result<String> {
+    let credential_path = format!("{}{}", move_home, MOVEY_CREDENTIAL_PATH);
```

Comment:
This PR has similar Window compatibility problems as #226 which should be addressed.